src/ConstructiveSolidGeometry/VolumePrimitives/VolumePrimitives.jl

```diff
@@ -1,5 +1,19 @@
-include("Cone.jl")
+ClosedPrimitive(p::VP) where {VP <:AbstractVolumePrimitive} = VP(p, COT = ClosedPrimitive)
+OpenPrimitive(p::VP) where {VP <: AbstractVolumePrimitive} = VP(p, COT = OpenPrimitive)
```

Comment:
Never mind, I see it now why you chose `COT`.
I would resolve all comments regarding this as it is easier than going back and finding them.. Sorry.